c2corg_ui/templates/article/helpers/view.html

```diff
@@ -31,10 +31,49 @@
       <p><span class="value-title" translate>article_type</span>: <span class="value" x-translate>${article['article_type']}</span></p>
     % endif
 
+    % if article.get('author') and article['article_type'] != 'collab':
+      <%
+          user_id = article['author']['user_id']
+          user_lang = article['locales'][0]['lang']
+      %>
+      <p><span class="value-title" translate>author</span>:
+        <a class="value" href="${request.route_path('profiles_view', id=user_id, lang=user_lang)}" >${article['author']['name']}</a>
+    % endif
+
     % if article.get('quality'):
       <p><span class="value-title" translate>quality</span>: <span class="value" x-translate>${article['quality']}</span></p>
     % endif
 
     </span>
 </div>
 </%def>
+
+<%def name="get_article_participants(article)">\
+  % if article.get('participant_count') or article['locales'][0].get('participants') or article['associations']['users']:
+    % if len(article['associations']['users']) > 1 or not article['associations']['users'][0]['document_id'] == article['author']['user_id']:
+      <div class="name-icon-value" ng-click="detailsCtrl.openTab($event)">
+        <h4><span translate>associated users</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+        <div class="name-icon">
+          <span class="icon-user"></span>
+          <p translate>associated users</p>
+        </div>
+        <span class="detail-text accordion">
+          % if article.get('associations').get('users'):
+            <p>
+              % for user in article['associations']['users']:
+                <%
+                    user_id = user['document_id']
+                    user_lang = user['locales'][0]['lang']
+                    is_last_participant = loop.last and not article['locales'][0].get('participants')
```

Comment:
That is really little weird. I renamed it to assoc. users and cleaned code little bit.